src/webapp/app.py

```diff
@@ -241,16 +244,18 @@ def graph():
         # user's session. This shouldn't be cached across user sessions.
         http_cache: dict = session.get("msal_http_response_cache", {})
 
-        # Create an MSAL client (using app configuration values) and providing
-        # our token cache.
+        #<ms_docref_configure_app>
+        # Create an MSAL client using the app's configuration values and provide the token cache.
         msal_client = msal.ConfidentialClientApplication(
             app.config.get("CLIENT_ID"),
             authority=app.config.get("AUTHORITY"),
             client_credential=app.config.get("CLIENT_CREDENTIAL"),
             token_cache=token_cache,
             http_cache=http_cache,
         )
+        #</ms_docref_configure_app>
 
+        #<ms_docref_acquire_token>
```

Comment:
Style has been adding a space before them, but you have it mixed in here.  Can you make them all with a leading space like this example?

```suggestion
        # </ms_docref_configure_app>

        # <ms_docref_acquire_token>
```